docs/authentication/social-connections/oauth.mdx

```diff
@@ -102,39 +120,35 @@ export async function GET() {
 }
 ```
 
-## Add social connection after sign-up
+## Add a social connection after sign-up
 
-For each OAuth provider, you can disable the option to sign up and sign in to your application using the provider. This is especially useful for users that want to connect their OAuth account after authentication.
+For each social provider, you can disable the option to sign up and sign in to your application using the provider. This is especially useful for users that want to connect their OAuth account _after_ authentication.
 
 For example, say your application wants to read a user's GitHub repository data but doesn't want to allow the user to authenticate with their GitHub account. The user can sign up with their email and password, or whatever method you choose, and then afterwards, connect their GitHub account to your application.
 
-After sign-up, connections can be made through our [`<UserProfile/>`](/docs/components/user/user-profile) component, or with a [custom flow](/docs/custom-flows/oauth-connections).
+After sign-up, connections can be made through Clerk's [`<UserProfile/>`](/docs/components/user/user-profile) component, or with a [custom flow](/docs/custom-flows/oauth-connections).
 
-To disable the option for users to sign up and sign in with a provider:
+To configure the option for users to sign up and sign in with a social provider:
 
 1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
 1. In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**.
-1. Next to the provider you want to configure, select the settings cog icon.
-1. Toggle off **Enable for sign-up and sign-in**.
+1. Select the social provider you want to configure.
```

Comment:
I think "cog" is a confusing term TBH, I might be wrong but as a non native English speaker, it took me a while to figure out the meaning.

Reply:
You don't need to select the settings cog icon. You can just select the provider. And for the same reason Laura was saying, I wanted to avoid using that kind of language in case its confusing - especially if it can be avoided like in this case.